### PR TITLE
Repository collection, text details, Concept list and details

### PR DESCRIPTION
**Pull request by nowke**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -73,4 +73,9 @@ export default Vue.extend({
 		}
 	}
 }
+.main {
```
Comment (tjquinn1) [inline src/App.vue:76]: Do you actually use this class?

File: src/views/concepts/ConceptList.vue [hunk 2]
```diff
@@ -0,0 +1,154 @@
+<template lang="pug">
```
Comment (tjquinn1) [inline src/views/concepts/ConceptList.vue:1]: This page renders but will not load anything.  I have attached screen shots of the errors

Screen Shot 2019-12-02 at 1 55 13 PM
Screen Shot 2019-12-02 at 1 55 20 PM

**Review by tjquinn1 — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 1 above)
Comment (nowke) [reply src/App.vue:76]: Yes. It is used in all pages as containers.

File: src/views/concepts/ConceptList.vue (hunk 2 above)
Comment (nowke) [reply src/views/concepts/ConceptList.vue:1]: Related to backend https://github.com/diging/vogon-web/pull/155

**Review by tjquinn1 — APPROVED** [review]

**Pull request merged**